### PR TITLE
Frigid & 2G2C Entries

### PR DESCRIPTION
**Pull request by Just-a-Unity-Dev**
Makes an entry for Frigid and 2G2C (dead forks are allowed in the DB, right?)

### CODE REVIEW

File: WizardsDen/Children/2G2C/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+| Key  | Value |
+| ------------- | ------------- |
+| Host(s) | `DSC@eclips_e#0001/GH@Just-a-Unity-Dev` |
+| Repository  | https://github.com/Spesships/2G2C |
+| Discord  | https://discord.gg/n8se25bGCx |
+| Website | https://421034.xyz/ <!-- (domain looks weird, but I own it and it's cheap af) --> |
+| Language | English |
+| Info | An NRP anarchy server (except without the slurs) |
+| Date of Creation | Roughly around 5/29/2022 |
+| Date of End | 9/3/2022 |
```
Comment (mirrorcult) [inline WizardsDen/Children/2G2C/README.md:14]: ISO-8601 your dates (YYYY-MM-DD)

File: WizardsDen/Children/Frigid/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+| Key  | Value |
+| ------------- | ------------- |
+| Host(s) | `DSC@eclips_e#0001/GH@Just-a-Unity-Dev` |
+| Repository  | https://github.com/2G2C/frigid |
+| Discord  | https://discord.gg/n8se25bGCx |
+| Website | https://421034.xyz/ <!-- (domain looks weird, but I own it and it's cheap af) --> |
+| Language | English |
+| Info | A MRP fork server focusing around the concept of zombies and AI |
+| Date of Creation | 9/3/2022 (GMT+8) |
```
Comment (mirrorcult) [inline WizardsDen/Children/Frigid/README.md:13]: no timezone info needed

File: WizardsDen/Children/Frigid/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Frigid
+
+<img src="frigid-logo.png" width=300>
```
Comment (mirrorcult) [inline WizardsDen/Children/Frigid/README.md:3]: rename file to just `logo.png`

**Review by mirrorcult — CHANGES_REQUESTED** [review]
archive https://421034.xyz/ if it is not already done

File: WizardsDen/Children/Frigid/README.md (hunk 3 above)
Comment (Just-a-Unity-Dev) [reply WizardsDen/Children/Frigid/README.md:3]: > archive https://421034.xyz/ if it is not already done

recycling the same domain for frigid (unless i misread it)

Comment (Just-a-Unity-Dev) [reply WizardsDen/Children/Frigid/README.md:3]: what why did that go in the review not comments

i hate github mobile

**Comment by Just-a-Unity-Dev** [comment]
> archive https://421034.xyz/ if it is not already done

i'm recycling the same domain for frigid (unless i misread, if i did feel free to clarify)

**Comment by Just-a-Unity-Dev** [comment]
reviews addressed

File: WizardsDen/Children/2G2C/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+| Key  | Value |
+| ------------- | ------------- |
+| Host(s) | `DSC@eclips_e#0001/GH@Just-a-Unity-Dev` |
+| Repository  | https://github.com/Spesships/2G2C |
+| Discord  | https://discord.gg/n8se25bGCx |
+| Website | [421034.xyz](http://web.archive.org/web/20220907144712/https://421034.xyz/) <!-- (domain looks weird, but I own it and it's cheap af) --> |
+| Language | English |
+| Info | An NRP anarchy server (except without the slurs) |
+| Date of Creation | Around 2022/05/29 |
+| Date of End | 2022/09/03 |
```
Comment (mirrorcult) [inline WizardsDen/Children/2G2C/README.md:14]:
```suggestion
| Date of Creation | Around 2022-05-29 |
| Date of End | 2022-09-03 |
```

File: WizardsDen/Children/Frigid/README.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+| Key  | Value |
+| ------------- | ------------- |
+| Host(s) | `DSC@eclips_e#0001/GH@Just-a-Unity-Dev` |
+| Repository  | https://github.com/2G2C/frigid |
+| Discord  | https://discord.gg/n8se25bGCx |
+| Website | [421034.xyz](https://421034.xyz/) <!-- (domain looks weird, but I own it and it's cheap af) --> |
+| Language | English |
+| Info | A MRP fork server focusing around the concept of zombies and AI |
+| Date of Creation | 2022/09/03 |
```
Comment (mirrorcult) [inline WizardsDen/Children/Frigid/README.md:13]:
```suggestion
| Date of Creation | 2022-09-03 |
```

**Review by mirrorcult — CHANGES_REQUESTED** [review]

**Pull request merged**